src/sidebar/components/TopBar.tsx

```diff
@@ -108,28 +104,32 @@ function TopBar({
               />
               <SortMenu />
               {showShareButton && (
-                <IconButton
+                <ToggleIconButton
                   icon={ShareIcon}
                   expanded={isAnnotationsPanelOpen}
+                  pressed={isAnnotationsPanelOpen}
                   onClick={toggleSharePanel}
                   size="xs"
                   title="Share annotations on this page"
+                  data-testid="share-icon-button"
                 />
               )}
             </>
           )}
-          <IconButton
+          <ToggleIconButton
             icon={HelpIcon}
             expanded={isHelpPanelOpen}
+            pressed={isHelpPanelOpen}
             onClick={requestHelp}
             size="xs"
             title="Help"
+            data-testid="help-icon-button"
           />
           {isLoggedIn ? (
             <UserMenu onLogout={onLogout} />
           ) : (
             <div
-              className="flex items-center text-md font-medium space-x-1"
+              className="flex items-center text-md font-medium space-x-1 pl-1"
```

Comment:
The rest of elements in the top bar have a "double" spacing due to the padding on every button. Adding this extra left padding here to make it consistent, as it's specially important now that the buttons can render a border when pressed.

This is how it looked before (it's subtle, but you'll notice the links are closer to the previous button than the rest of the buttons are between themselves).

![image](https://github.com/hypothesis/client/assets/2719332/74de64f2-344c-4927-a64e-70b151259ca5)

And this is how it looks now, when previous button is unpressed and pressed.

![Captura desde 2023-11-29 14-25-25](https://github.com/hypothesis/client/assets/2719332/83a73f32-038d-4895-93a7-dbce083c66f1)

![Captura desde 2023-11-29 14-24-04](https://github.com/hypothesis/client/assets/2719332/2addb2a8-ee41-4cd9-9724-9839bf9a2d67)